README.md

```diff
@@ -1,4 +1,8 @@
 # Helm Resources for WSO2 Kubernetes Pipeline
+> **Warning**
+>
+>   This development has been suspended, and the CICD pipeline is deprecated.
```

Comment:
what is `this`? can we word it better? 

should we say, `has been suspended ... `?

Reply:
This is changed now, please check